src/main/java/com/lorepo/icplayer/client/module/text/AudioButtonWidget.java

```diff
@@ -133,6 +133,17 @@ public String getId() {
         return null;
     }
 
+    @Override
+	public boolean getResetStatus() {
+        // 	not implement
+        return false;
+	}
+
+	@Override
+	public void setResetStatus(boolean wasReset) {
+        // 	not implement
+	}
+
     @Override
     public void setFocusGap(boolean focus) {
         if (focus) {
```

Comment:
W nawiązaniu do naszej rozmowy należy zmienić nazwę komentarza na `TODO not implement`